### PR TITLE
Add release notes for NSS 3.63.1 and 3.64

### PR DESCRIPTION
**Pull request by beurdouche**

### CODE REVIEW

File: files/en-us/mozilla/projects/nss/nss_3.64_release_notes/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: NSS 3.64 release notes
+slug: Mozilla/Projects/NSS/NSS_3.64_release_notes
+---
+<h2 id="Introduction">Introduction</h2>
+
+<p>Network Security Services (NSS) 3.64 was released on <strong>18 March 2021</strong>.</p>
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/projects/nss/nss_3.64_release_notes/index.html:7]: The releases landing page says 15 April 2021; date needs updating here?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@beurdouche looking mostly good; just one comment to address. Thanks!

File: files/en-us/mozilla/projects/nss/nss_3.64_release_notes/index.html (hunk 1 above)
Comment (beurdouche) [reply files/en-us/mozilla/projects/nss/nss_3.64_release_notes/index.html]: Nice catch ! Updated. Thanks Chris : )

**Review by chrisdavidmills — APPROVED** [review]
Perfect, thanks @beurdouche !

**Pull request merged**